k8s/infrastructure.yaml

```diff
@@ -534,6 +535,8 @@ spec:
           - 's4-subscription-manager'
           - '--domain'
           - '$(S4_DOMAIN)'
+          - '--cross-domain'
+          - '$(CROSS_DOMAIN)'
```

Comment:
fwiw the only reason the parameter and its value are split into separate strings here is that I was just learning about how kubernetes env var interpolation works when I wrote this section.  I wasn't sure it was allowed to interpolate into a string like `--cross-domain=$(CROSS_DOMAIN)` but as you can see above that style does work and I've used it elsewhere.

Doesn't really make any difference, just explaining why there are two different styles in this file.